designer/client/src/actions/notificationActions.tsx

```diff
@@ -1,29 +1,48 @@
 import React from "react";
 import Notifications from "react-notification-system-redux";
-import TipsError from "../assets/img/icons/tipsError.svg";
-import TipsInfo from "../assets/img/icons/tipsInfo.svg";
-import TipsSuccess from "../assets/img/icons/tipsWarning.svg";
+import CheckCircleIcon from "@mui/icons-material/CheckCircle";
+import ErrorIcon from "@mui/icons-material/Error";
+import InfoIcon from "@mui/icons-material/Info";
 import Notification from "../components/notifications/Notification";
 import { Action } from "./reduxTypes";
 
 export function success(message: string): Action {
     return Notifications.success({
         autoDismiss: 10,
-        children: <Notification icon={<TipsSuccess />} message={message} />,
+        children: (
+            <Notification
+                type={"success"}
+                icon={<CheckCircleIcon sx={{ color: "#333333", alignSelf: "center" }} fontSize="inherit" />}
+                message={message}
+            />
+        ),
     });
 }
 
 export function error(message: string, error?: string, showErrorText?: boolean): Action {
     const details = showErrorText && error ? error : null;
     return Notifications.error({
         autoDismiss: 10,
-        children: <Notification icon={<TipsError />} message={message} details={details} />,
+        children: (
+            <Notification
+                type={"error"}
+                icon={<ErrorIcon sx={{ color: "#333333", alignSelf: "center" }} fontSize="inherit" />}
```

Comment:
Since the style of the icon is the same in the case of all notification variants, can we move `{ color: "#333333", alignSelf: "center" }` style inside of the `Notification` component? We can probably overwrite one of the Alert classes https://mui.com/material-ui/api/alert/#css

Reply:
done